doc/concept/granular-certificates/attributes.md

```diff
@@ -19,19 +19,17 @@ It consist of two parts
 
 A GSRN (Global Service Relation Number) is an unique identifier for a meter.
 
-To ensure the privacy and hide the relations between GCs the GSRN value is hidden with the help
-of a [Pedersen Commitment](../pedersen-commitments.md)
+To ensure the privacy and hide the relations between GCs the GSRN value is hidden using a [Pedersen Commitment](../pedersen-commitments.md)
```

Comment:
The GSRN are hashed because it is static data and would not benefit from a Pedersen Commitment